src/main.c

```diff
@@ -51,13 +51,15 @@ const char args_doc[] = "";
 
 static struct argp_option options[] = {
 	{ "debug", 'd', 0, OPTION_ARG_OPTIONAL, "Display debug/verbose output" },
+	{ "desktop", 'D', 0, OPTION_ARG_OPTIONAL, "Treat system as a desktop platform" },
```

Comment:
Would be nice to have a flag to treat the system as a server platform.

Reply:
I could but I really want to keep the assumption that things are a server platform and we optimize for that. Hence, the behavior is server-default everything. Hence, such a cmdline flag would be a no-op.